cocos/core/components/model-renderer.ts

```diff
@@ -49,9 +49,23 @@ export class ModelRenderer extends Renderer {
         this._onVisibilityChange(val);
     }
 
+    /**
+     * @en The transparent sorting priority which will be applied to the committed models.
+     * @zh 应用于所有提交渲染的 Model 在透明队列中的排序优先级
+     */
+    get transparentPriority () {
```

Comment:
priority is priority, it shouldn't be coupled with transparent or opaque queue

Reply:
I remember there was sorting priority added or is it the same PR ?

Reply:
This value is not meaningful for opaque queues. the opaque queue has depth writing turned on, so no matter what order is used to render, the ones closer to the screen are rendered to the front, so this value only works for the translucent queue